common/cmake/devices.cmake

```diff
@@ -134,6 +136,13 @@ function(DEFINE_ARCH)
   # * OUT_PLACE - VPR .place file to be used by router
   # * VPR_ARCH  - Path to VPR architecture XML file
   #
+  # SDC_PATCH_TOOL allows to specify a utility that processes SDC constraints
+  #  file provided by a user prior to feedin it to VPR.
+  #
+  # SDC_PATCH_TOOL_CMD variables:
+  #
+  # * TBD.
```

Comment:
Probably missing comment description of the `SDC_PATCH_TOOL_CMD` variables

Reply:
Comments added.